### PR TITLE
ci: Record primary context results using g-d-t-r --report-directory

### PR DESCRIPTION
**Pull request by cgwalters**
So the output isn't all intermingled.  Yes, need to improve g-d-t-r.

### CODE REVIEW

**Comment by cgwalters** [comment]
Blah, really just need to fix g-d-t-r to have a saner option to record the output even for successful tests.

**Comment by cgwalters** [comment]
> Removing gdtr-results/

Ahhh right.

File: ci/build-check.sh [hunk 1]
```diff
@@ -25,3 +39,11 @@ if test -x /usr/bin/clang; then
     export CC=clang
     build
 fi
+
+# Keep this in sync with papr.yml
+# TODO; Split the main/clang builds into separate build dirs
+for x in test-suite.log config.log gdtr-results; do
+    if test -e ${x}; then
```
Comment (jlebon) [inline ci/build-check.sh]: This should be `test -e ${resultsdir}/${x}` right?

**Comment by jlebon** [comment]
Looks like it worked! Though that last fixup is still targeting the wrong commit.

**Comment by cgwalters** [comment]
"I know, I'll use regular expressions!" ... https://github.com/cgwalters/homegit/commit/db05a911c0c73e051048c847f2155940a42dcac6

**Comment by jlebon** [comment]
@rh-atomic-bot r+ ee61d31

**Pull request closed without merging**